SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -230,6 +232,21 @@ void USpatialReceiver::HandleActorAuthority(Worker_AuthorityChangeOp& Op)
 		return;
 	}
 
+	if (Op.component_id == SpatialConstants::CLIENT_RPC_ENDPOINT_COMPONENT_ID
+		&& Op.authority == WORKER_AUTHORITY_AUTHORITATIVE)
+	{
+		ensure(!NetDriver->IsServer());
```

Comment:
I think this should be a `check` over an `ensure`.

Reply:
If this ever happens there are a lot bigger issues.

Reply:
Changed to check.